android/filament-android/src/main/cpp/filament-js/jsbindings.cpp

```diff
@@ -0,0 +1,943 @@
+        return KtxUtility::toPixelDataType(self->getInfo());
+    }), allow_raw_pointers())
+
+    /// getCompressedPixelDataType ::method::
+    /// ::retval:: [CompressedPixelDataType]
+    /// Returns "undefined" if no valid Filament enumerant exists.
+    .function("getCompressedPixelDataType",
+            EMBIND_LAMBDA(driver::CompressedPixelDataType, (KtxBundle* self), {
+        return KtxUtility::toCompressedPixelDataType(self->getInfo());
+    }), allow_raw_pointers())
+
+    /// isCompressed ::method::
+    /// Per spec, compressed textures in KTX always have their glFormat field set to 0.
+    /// ::retval:: boolean
+    .function("isCompressed", EMBIND_LAMBDA(bool, (KtxBundle* self), {
+        return KtxUtility::isCompressed(self->getInfo());
+    }), allow_raw_pointers())
+
+    .function("isCubemap", &KtxBundle::isCubemap)
+    .function("_getBlob", EMBIND_LAMBDA(BufferDescriptor, (KtxBundle* self, KtxBlobIndex index), {
+        uint8_t* data;
+        uint32_t size;
+        self->getBlob(index, &data, &size);
+        return BufferDescriptor(data, size);
+    }), allow_raw_pointers())
+    .function("_getCubeBlob", EMBIND_LAMBDA(BufferDescriptor,
+            (KtxBundle* self, uint32_t miplevel), {
+        uint8_t* data;
+        uint32_t size;
+        self->getBlob({miplevel}, &data, &size);
+        return BufferDescriptor(data, size * 6);
+    }), allow_raw_pointers())
+
+    /// info ::method:: Obtains properties of the KTX header.
+    /// ::retval:: The [KtxInfo] property accessor object.
+    .function("info", &KtxBundle::info)
+
+    /// getMetadata ::method:: Obtains arbitrary metadata from the KTX file.
+    /// key ::argument:: string
+    /// ::retval:: string
+    .function("getMetadata", EMBIND_LAMBDA(std::string, (KtxBundle* self, std::string key), {
+        return std::string(self->getMetadata(key.c_str()));
+    }), allow_raw_pointers());
+
+function("KtxUtility$createTexture", EMBIND_LAMBDA(Texture*,
+        (Engine* engine, const KtxBundle& ktx, bool srgb, bool rgbm), {
+    return KtxUtility::createTexture(engine, ktx, srgb, rgbm, nullptr, nullptr);
+}), allow_raw_pointers());
+
+/// KtxInfo ::class:: Property accessor for KTX header.
+/// For example, `ktxbundle.info().pixelWidth`. See the
+/// [KTX spec](https://www.khronos.org/opengles/sdk/tools/KTX/file_format_spec/) for the list of
+/// properties.
+class_<KtxInfo>("KtxInfo")
+    .property("endianness", &KtxInfo::endianness)
+    .property("glType", &KtxInfo::glType)
+    .property("glTypeSize", &KtxInfo::glTypeSize)
+    .property("glFormat", &KtxInfo::glFormat)
+    .property("glInternalFormat", &KtxInfo::glInternalFormat)
+    .property("glBaseInternalFormat", &KtxInfo::glBaseInternalFormat)
+    .property("pixelWidth", &KtxInfo::pixelWidth)
+    .property("pixelHeight", &KtxInfo::pixelHeight)
+    .property("pixelDepth", &KtxInfo::pixelDepth);
+
+register_vector<std::string>("RegistryKeys");
+
+/*
```

Comment:
I commented this out because of missing link depeendencies. should be easier to solve now.